docs/data-sources/lb_certificate.md

```diff
@@ -47,14 +47,14 @@ data "scaleway_lb_certificate" "byName" {
 
 The following arguments are supported:
 
-- `certificate_id` - (Optional) The certificate id.
+- `certificate_id` - (Optional) The certificate ID.
     - Only one of `name` and `certificate_id` should be specified.
 
 - `name` - (Optional) The name of the certificate backend.
```

Comment:
@yfodil Should this be "frontend", since certificates are not attached to a backend, but a frontend? It's not very clear to me

Reply:
Yes certs are bind to frontends, maybe change it to something like:
```suggestion
- `name` - (Optional) The name of the Load Balancer certificate.
```